packages/orchestrator-ui-components/src/components/WfoSubscription/WfoSubscriptionProductBlock/WfoSubscriptionProductBlock.tsx

```diff
@@ -188,39 +160,10 @@ export const WfoSubscriptionProductBlock = ({
                                         ),
                                 )
                                 .map((productBlockInstanceValue, index) => (
-                                    <tr key={index}>
```

Comment:
not part of your PR but I would add the  `productBlockInstanceValues.filter()` above as a const outside the html